bluesky/run_engine.py

```diff
@@ -91,6 +91,10 @@ def __set__(self, obj, value):
      Parameter('subs', Parameter.POSITIONAL_ONLY, default=None),
      Parameter('metadata_kw', Parameter.VAR_KEYWORD)])
 
+#Default scan_id_source
```

Comment:
This comment can probably be removed because it doesn't convey any information that is not already conveyed by the function ame.

Reply:
Removed